Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py

```diff
@@ -102,6 +105,9 @@ def create_indicator(self, item_type, value, mitre_item_json):
         }
 
         indicator_obj['fields']['tags'].extend(self.tags)
+        tlp = indicator_obj['fields']['tlp']
+        if tlp != '':
+            indicator_obj['fields']['trafficlightprotocol'] = tlp
```

Comment:
Currently the integration set `TLP` will override the `TLP` provided in the data of the indicator, please confirm with @Ni-Knight that the integration parameter `TLP` should override the `TLP` from the data.
If that's the case, then we can skip the `TLP` extraction when `self.tlp` is set - or better yet, return the configured value in `get_tlp` (so we only have one place in the code where we get the tlp value and only one place where we set the tlp value).

Reply:
I missed the elif here.
You are right. 
I have talked with @Ni-Knight about it. If we extract a `tlp` then we use it otherwise use the `self.tlp`